src/mlpack/core/dists/discrete_distribution.hpp

```diff
@@ -189,8 +189,8 @@ class DiscreteDistribution
    * in logProbabilities.
    *
    * @param x List of observations.
-   * @param logProbabilities probabilities Output probabilities for each 
-   *   input observation.
+   * @param logProbabilities Probabilities Output probabilities for each 
```

Comment:
I think it should just be `Output probabilities...` in place of `Probabilities Output probabilities...`

Reply:
Agreed, really thanks @ShikharJ :)